pandas/core/groupby/grouper.py

```diff
@@ -284,7 +284,7 @@ def __init__(
             if self.name is None:
                 self.name = index.names[level]
 
-            self.grouper, self._labels, self._group_index = index._get_grouper_for_level(
+            self.grouper, self._labels, self._group_index = index._get_grouper_for_level(  # noqa
```

Comment:
this is a case where black produces too long line lengths (the original code before black was using `\` after `=` for line continuation, but checked, and black always removes this when adding it back.

For now just adding `# noqa` for this and similar cases (long term the code could be refactored a bit too avoid this, but now just fixing flake8)

Reply:
Makes sense to me!

Reply:
Actually can you add `# noqa: E501` to these? I suppose makes sense to only filter the line length error here